data/python/tests/test_jepbridge.py

```diff
@@ -28,10 +28,10 @@ def test_type_instance(self):
         from java.util import Date
         from ghidra.program.database import ProgramDB
 
-        self.assertIsInstance(Date(), Object.__pytype__)
-        self.assertIsInstance(Date(), Serializable.__pytype__)
-        self.assertTrue(issubclass(Date.__pytype__, Object.__pytype__))
-        self.assertTrue(issubclass(Date.__pytype__, Serializable.__pytype__))
```

Comment:
Was `__pytype__` removed in the latest `Jep` release? If not, let's keep these tests for good measure. If `__pytype__` was removed then let's be sure to de-duplicate these tests.

Reply:
@mike-hunhoff I've confirmed that the latest `Jep` release still has the `__pytype__` access in their unit test. I'll be adding these back in the next commit and updating the documentation.